daemon/logger/bufferlog/bufferlog.go

```diff
@@ -0,0 +1,93 @@
+package bufferlog
+
+import (
+	"github.com/alibaba/pouch/daemon/logger"
+	rb "github.com/alibaba/pouch/pkg/ringbuffer"
+
+	"github.com/docker/go-units"
+	"github.com/sirupsen/logrus"
+)
+
+// BufferLog is uses to cache the container's logs with ringBuffer.
+type BufferLog struct {
+	ringBuffer *rb.RingBuffer
+	logger     logger.LogDriver
+}
+
+// NewBufferLog return a new BufferLog.
+func NewBufferLog(logDriver logger.LogDriver, info logger.Info) (logger.LogDriver, error) {
+	bufferSize := int64(-1)
```

Comment:
Just curious, why should you define the `bufferSize` to `int`? 

So, you no need covert the `int64` to `int` in line 31